src/storage/txn/commands/check_txn_status.rs

```diff
@@ -225,20 +227,17 @@ pub mod tests {
             resolving_pessimistic_lock,
             deadline: Deadline::from_now(DEFAULT_EXECUTION_DURATION_LIMIT),
         };
-        assert!(
-            command
-                .process_write(
-                    snapshot,
-                    WriteContext {
-                        lock_mgr: &DummyLockManager,
-                        concurrency_manager: cm,
-                        extra_op: Default::default(),
-                        statistics: &mut Default::default(),
-                        async_apply_prewrite: false,
-                    },
-                )
-                .is_err()
+        let res = command.process_write(
+            snapshot,
+            WriteContext {
+                lock_mgr: &DummyLockManager,
+                concurrency_manager: cm,
+                extra_op: Default::default(),
+                statistics: &mut Default::default(),
+                async_apply_prewrite: false,
+            },
         );
+        res.map(|_| ()).unwrap_err()
```

Comment:
🤔 What does the `map(|_| ())` part do?

Reply:
`unwrap_err` will print the `Ok()` varient when it panics. But `WriteContext` is not `Debug`, so it will not compile. So I use `.map(|_| ())` to workaround.